docs/structured-streaming-kafka-integration.md

```diff
@@ -499,20 +522,28 @@ StreamingQuery ds = df
   .option("topic", "topic1")
   .start();
 
-// Write key-value data from a DataFrame to Kafka using a topic specified in the data
+// Write key-value data from a DataFrame to Kafka using the topic specified in the data
 StreamingQuery ds = df
   .selectExpr("topic", "CAST(key AS STRING)", "CAST(value AS STRING)")
   .writeStream()
   .format("kafka")
   .option("kafka.bootstrap.servers", "host1:port1,host2:port2")
   .start();
 
+// Write key-value data from a DataFrame to Kafka using the topic specified in the path option
+StreamingQuery ds = df
```

Comment:
nit: The name `ds` is not consistent with the type which is `StreamingQuery`. Please use `sq` or `q` instead.